<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormsDialog</class>
 <widget class="QDialog" name="FormsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="fillColorButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>select fill color</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>167</width>
     <height>29</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Fill Form</string>
   </property>
  </widget>
  <widget class="QPushButton" name="strokeColorButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>select stroke color</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="rectanglePage">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>122</width>
       <height>88</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="rectangleWidthSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="rectangleHeightSpinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="circlePage">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Radius</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="circleSpinBox"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="trianglePage">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>122</width>
       <height>109</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>side A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="triSideASpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>side B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="triSideBSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>side C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="triSideCSpinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>select form</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="formsComboBox">
      <item>
       <property name="text">
        <string>Rectangle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Circle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Triangle</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
